ivadomed/transforms.py

```diff
@@ -474,7 +486,7 @@ def __call__(self, sample):
                 for idx, gt in enumerate(labels):
                     pil_img = Image.fromarray(gt, mode='F')
                     gt_vol[idx, :, :] = np.array(self.sample_augment(pil_img, params))
-                ret_gt.append(gt_vol.astype('float32'))
```

Comment:
You may have corrected this in a latter commit --> but we want to keep float32 when using this transform because we are modifying the GT: GT_voxels=1 and new surrounding voxels € ]0, 1[.

Reply:
Yes, it was corrected :)